closed/test/jdk/openj9/internal/security/TestProperties.java

```diff
@@ -43,6 +43,8 @@
 
 public class TestProperties {
 
+    static String fileSep = java.io.File.pathSeparator;
```

Comment:
This is a misleading name; suggest just using `File.pathSeparator` directly instead.

Reply:
Updated.